content/en/funding/community-projects/index.mdx

```diff
@@ -1,97 +1,61 @@
 ---
-description: If you’ve got an idea for improving the MakerDAO community coordination or resources, we can provide up to 5,000 Dai to help make it happen.
+title: Community Projects
+keywords: Autonomous, Single Owner, Projects
 ---
 
-# Community Projects
+# Individual Contributions and Projects
```

Comment:
I don't think "Individual Contributions" makes sense in this context. I recommend we stick to Community Projects for V1 or "Community Initiative Proposals" or "Community Initiative Projects"